src/pages/Event.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import EventPage from '../components/events/event/EventPage/EventPage';
+import EventPage from '../components/events/Event/EventPage/EventPage';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../components/events/Event/EventPage/EventPage'.